tests/transforms/test_batch_inputs.py

```diff
@@ -1,129 +0,0 @@
-# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
```

Comment:
How come this file is being deleted? (Not sure why there's a separate `tests/transforms/test_batch_input.py` and `tests/transforms/test_batch_inputs.py` file anyways :thinking: )

Reply:
They have the same contents, that's why I suggest to delete this one. I noticed because the fix in `test_batch_input.py` would have been required here as well.